management/server/http/api/openapi.yml

```diff
@@ -369,6 +369,8 @@ components:
         expires_in:
           description: Expiration time in seconds
           type: integer
+          minimum: 86400
+          maximum: 31536000
           example: 43200
```

Comment:
minor change. Let's update the `expires_in` seconds in the example, ensuring compliance with the minimum allowed value of 86400 seconds.